Detectors/ITSMFT/common/reconstruction/include/ITSMFTReconstruction/PixelData.h

```diff
@@ -84,8 +84,13 @@ class PixelData
 
  private:
   void sanityCheck() const;
+#ifdef ENABLE_UPGRADES
+  static constexpr int RowMask = 0x7FFF; ///< 32768 rows are supported
+  static constexpr int MaskBit = 0x8000; ///< 16-th bit is used to flag masked pixel
+#else
```

Comment:
Since everything else is a copy-paste from current ITS, commenting only here: I would simply remove the ifdef and leave IT3 masks, it is anyway there to account for the masking bit. As far as both ITS2 and ITS3 rows fit to 15 bits, we don't need to differentiate them

Reply:
OK! Fixed, thanks!